### ISSUE TITLE
[Discussion] Update email address

### ISSUE THREAD

**Issue by akwick** [gophers-frm/web#6]
Should we replace the current Gmail address with our new organizers' address? The advantage is that we share one mail address consistently and the disadvantage I see is that the address will be in the YAML file. 

Do we want to add our firstname@ addresses to about?

**Comment by arnehormann** [comment]
All our email addresses are visible by reading git commits in our repos - so not really something hidden. But not visible to simplistic web-scrapers, which is nice. I prefer adding the single address here.

**Comment by sascha-andres** [comment]
I agree with @arnehormann. I think we the "group" mail for public incoming communication is a good use case. Also so, that we all can react to inquiries sent using the organizers mail.

**Comment by akwick** [comment]
- [ ] Update the email address
